core/src/test/scala/shapeless/sybclass.scala

```diff
@@ -22,23 +22,30 @@ import org.junit.Assert._
 import test._
 
 class SybClassTests {
-  object gsizeAll extends Poly1 {
-    implicit def caseString = at[String](_.length)
+
+  trait gsizeAll0 extends Poly1 {
```

Comment:
Is this name ok? I chose this among `gsizeAllLowPriority`, `gsizeDefault` and PascalCased alternatives.